jmclient/jmclient/blockchaininterface.py

```diff
@@ -25,12 +26,11 @@ class BlockchainInterface(object):
     def __init__(self):
         pass
 
+    @abc.abstractmethod
     def is_address_imported(self, addr):
-        try:
-            return self.rpc('getaccount', [addr]) != ''
-        except JsonRpcError:
-            return len(self.rpc('getaddressinfo', [addr])['labels']) > 0
+        """checks that address is already imported"""
```

Comment:
I understand the normalization of this so it is like the other abstract methods, however: the concept of 'address imported' is very likely to not apply to any other future blockchain interfaces we implement. Notice the same is true of `import_addresses_if_needed`.

Since we've basically abandoned the alternative ElectrumInterface for now, and since after this PR, we intend to have encapsulated all the blockchain specific functionality inside the bci module (notice how previously, we had to add dummy import methods in ElectrumInterface, because there were external callers (wallet sync iirc) using them), it might seem logical to remove these methods entirely from the base class.

However we can see that wallet sync methods defined in `WalletService` are still using implementation-specific details like this; for example, `WalletService.sync_addresses()` is calling `import_addresses_if_needed`, etc. So for now probably do what you do here: enforce the existence of these methods with `abc.abstractmethod` and also add it for the other method, `import_addresses_if_needed`.